torchft/multi_quorum_test.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
```

Comment:
could move this test to `lighthouse_test.py` to keep it with the rest of the lighthouse tests